From 208bb414553d5444d82601e6fd4ca25fbb192998 Mon Sep 17 00:00:00 2001
From: Michal Sekletar <msekleta@redhat.com>
Date: Mon, 29 Sep 2014 08:19:05 +0200
Subject: [PATCH 1/4] man: tcpdump and tcpslice have manpages in man8

Both should be run only by root, hence manpages for them should be located in
/usr/share/man/man8/
---
 pcap.3pcap.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/pcap.3pcap.in
+++ b/pcap.3pcap.in
@@ -1009,8 +1009,8 @@
 1.0 APIs are available and use them only if they are.
 .SH SEE ALSO
 .BR autoconf (1),
-.BR tcpdump (1),
-.BR tcpslice (1),
+.BR tcpdump (8),
+.BR tcpslice (8),
 .BR \%pcap-filter (@MAN_MISC_INFO@),
 .BR bpf (@MAN_DEVICES@),
 .BR usermod (@MAN_ADMIN_COMMANDS@)
